Source/i_video.c

```diff
@@ -1208,6 +1208,9 @@ static void I_InitGraphicsMode(void)
                                SDL_TEXTUREACCESS_STREAMING,
                                v_w, v_h);
 
+   // Workaround for SDL 2.0.14 alt-tab bug (taken from Doom Retro)
```

Comment:
Doesn't this only occur in Windows? (This isn't a rhetorical question. I really don't know, haven't upgraded my SDL2 packages on Debian yet).

Isn't this workaround only required with SDL 2.0.14?

Reply:
I've only tested it on Windows, but probably yes to both questions.

Reply:
The issue is not that severe on Linux, i.e. the game remains fullscreen in the background, but at least I can change processes and get other windows into the foreground. So, this needs a build-time check for Windows and a run-time check for SDL 2.0.14, please. 😁

Reply:
Done.